public/app/features/correlations/CorrelationsPage.tsx

```diff
@@ -44,10 +45,30 @@ export default function CorrelationsPage() {
   const canWriteCorrelations = contextSrv.hasPermission(AccessControlAction.DataSourcesWrite);
 
   const handleAdd = useCallback(() => {
+    reportInteraction('grafana_correlations_added');
```

Comment:
nit: I'd change `handleAdd` to `handleAdded` just to make it clearer it's called after creating a new entry.

Reply:
updated in ba002760a1 (same for the update handler)